entries/slotfills/index.php

```diff
@@ -9,36 +9,35 @@
 
 namespace Alley\WP\WP_Curate;
 
-add_action(
-	'enqueue_block_editor_assets',
-	__NAMESPACE__ . '\action_enqueue_slotfills_assets'
-);
-
 /**
  * Registers all slotfill assets so that they can be enqueued through Gutenberg in
  * the corresponding context.
  */
 function register_slotfills_scripts(): void {
-	// Automatically load dependencies and version.
-	$asset_file = include __DIR__ . '/index.asset.php';
-
 	/**
 	 * Filter the post types that will show the "Enable Deduplication" slotfill.
+	 *
+	 * @param array $allowed_post_types The post types that will show the "Enable Deduplication" slotfill.
 	 */
 	$allowed_post_types = apply_filters( 'wp_curate_duduplication_slotfill_post_types', [ 'page', 'post' ] );
```

Comment:
I wonder why this is not falling back to the `$supported_post_types->get_supported_post_types()`. We can fallback to that and consumers would be able to override it here.